utils.js

```diff
@@ -8476,6 +8490,17 @@ export const getNugetMetadata = async function (
                 p.version +
                 "/"
             };
+            // license couldn't be properly identified and is still a url,
+            // therefore trying to resolve license via repository
+            if (
+              !p.license ||
+              (typeof p.license === "string" &&
+                p.license.startsWith("https://"))
+            ) {
+              p.license =
+                (await getRepoLicense(p.repository.url, undefined)) ||
```

Comment:
Does resolving license via git work for dotnet libraries?

Reply:
Yes, a lot of dotnet libraries are on GitHub. Even the newer versions of the .NET framework is open source on GitHub.